src/plugins/link-plugin.tsx

```diff
@@ -76,27 +77,33 @@ export function LinkPlugin(): HtmlSerializablePlugin {
         if (hasLink) {
           editor.command(unwrapLink);
         } else {
-          const textPrompt = value.selection.isExpanded ? [] : ["Enter the text for the link:"];
+          const textField: IField[] = value.selection.isExpanded
+                                        ? []
+                                        : [{ name: "linkText", type: "input",
+                                            label: "Enter the text for the link:" }];
+          const urlField: IField[] = [{ name: "linkUrl", type: "input",
+                                      label: "Enter the URL of the link:" }];
```

Comment:
We might want to unify the wording between the link and image plugins.  The link plugin uses "Enter the URL..." phrasing, but the image plugin now simply says "Source URL:".  Might be more clear if the wording was similar in the two.

Reply:
"Destination URL" or "Link URL"?

Reply:
I like Link URL.

Reply:
Done